packages/form-api/src/controllers/email.controller.ts

```diff
@@ -1,79 +1,32 @@
-                `When selecting your WorkBC Centre, select the community where your job is located.`,
-                `<img class="img-fluid" src="${s3static}/wage-subsidy-images/workbc_community_select.png" alt="WorkBC Community Selector" style="height: auto; line-height: 100%; outline: none; text-decoration: none; width: 100%; max-width: 100%; border: 0 none;">`,
-                `<b>Step 3:</b> Let your employer know you have applied! A team member will be in touch soon.`
-            ],
-            [
-                `<b>Why use WorkBC?</b></p><p>
-        <ul>
-          <li>
-            <b>Expertise: </b>We're ready to help you start career planning now or get you ready for the next phase of BC's COVID-19 Restart Plan.</li>
-          <li>
-            <b>Free Services: </b>We offer skills training and personalized, one-on-one job counselling. WorkBC services are completely free.</li>
-          <li>
-            <b>Benefits: </b>You might also be eligible for exclusive benefits.</li>
-        </ul>
-        `,
-                `Sincerely,<br><b>Your WorkBC team<br></b>`
-            ]
-        )
-    } else if (data.positionTitle0 && data.employeeEmail0 === undefined) {
-        emailHTML = generateHTMLEmail(
-            "Thank you, your application has been received",
-            [
-                `Thank you for your interest in WorkBC Wage Subsidy services.`,
-                `A WorkBC staff member will be in touch with you to confirm your business qualifies for WorkBC Wage subsidy, to complete the application process and will work with you on identifying a suitable match.`
-            ],
-            [],
-            []
-        )
-    } else {
-        emailHTML = generateHTMLEmail(
-            "A Wage Subsidy Claim Application has been submitted",
-            [
-                `<b>Catchment: ${data.container.workbcCentre.split("-")[0]}</b>`,
-                `Thank you for your interest in WorkBC Wage Subsidy services.`
-            ],
-            [],
-            []
-        )
+const createEmailHTMLBasedOnType = (applicationType: string) => {
+    let emailHTML = ""
+    if (String(applicationType) === "HaveEmployee") {
+        emailHTML = received.receivedHaveEmployee()
+    } else if (String(applicationType) === "needEmployee") {
+        emailHTML = received.receivedNeedEmployee()
     }
     return emailHTML
 }
 
-export const sendEmail = async (req: any, res: express.Response) => {
+export const sendEmail = async (req: express.Request, res: express.Response) => {
     try {
+        // console.log(req.body)
         const { data } = req.body
+        const applicationType = String(req.body.applicationType)
         let recipients: string[] = []
-        if (data.employeeEmail0 && data.employeeEmail0 !== null) {
+        if (String(applicationType) === "HaveEmployee") {
             recipients = Object.keys(data)
```

Comment:
Will this also obtain employee emails from the second position? Second position employee emails are stored in a nested object of data.